server/server.js

```diff
@@ -31,8 +40,11 @@ server.register([ inert, auth, ], (err) => {
   });
 
   server.route(routes);
+
 });
 
-socket(server.listener);
+createSocket(server.listener);
 
-module.exports = server;
+module.exports = {
```

Comment:
I know why we changed this originally, but it looks like `-module.exports = server;` would be ok now...